bioblend/cloudman/__init__.py

```diff
@@ -365,10 +365,17 @@ def __init__(self, url, password, **kwargs):
         else:
             super().__init__(None, None)
         self.config = kwargs.pop('cloudman_config', None)
+        self.use_ssl = False
         if not self.config:
             self.password = password
+            self.verify = kwargs.get("verify",None)
```

Comment:
I'd prefer to always have `self.verify` either True or False. For the rest of BioBlend we default to True (as in `requests.request()`, for better security), but we could default to False if it makes more sense for Cloudman.

Reply:
The issue is (and again, style and consistency matters, and I don't have enough context to choose wisely there) that you asked for verify to be only added for the get request if it was a https url (for http it is irrelevant). The None state allows us to set verify to True or False or to leave it out.

Reply:
The default launch process is to use a self-signed cert (because of DNS requirements) so `False` (vs. `True`) would provide more functional capability out of the box. Otherwise, personally, I don't have a strong preference between `False` or `None` given the above comment.

Reply:
> The issue is (and again, style and consistency matters, and I don't have enough context to choose wisely there) that you asked for verify to be only added for the get request if it was a https url (for http it is irrelevant).

I probably expressed myself badly, passing `verify` for HTTP requests is absolutely fine (as you said, it's irrelevant and ignored by `requests`).